### PR TITLE
Issue #TG 267 fix: method to close all Cassandra connection on server…

### PR DESCRIPTION
**Pull request by anoophm**
… close

### CODE REVIEW

File: server/src/api/index.ts [hunk 1]
```diff
@@ -47,8 +48,31 @@ export class FrameworkAPI {
...
+  closeCassandraConnections() {
+    let connectionsToBeClosedCount = this.cassandraConnection.length;
+    let cassandraConnection = this.cassandraConnection;
+    return new Promise(function (resolve, reject) {
+      logger.info('cassandra connectionsToBeClosedCount', connectionsToBeClosedCount);
+      if (!cassandraConnection.length) {
+        return resolve();
+      }
+      cassandraConnection.forEach(connection => {
+        connection.close(err => {
```
Comment (HarishGangula) [inline server/src/api/index.ts]: Log the error if there is any while closing connection

**Review by HarishGangula — CHANGES_REQUESTED** [review]
1. getConnectionByKeyspace is another method where the connection is open handle that

overall have the connections list inside cassandra/index.ts and expose that to api/index.ts

**Comment by anoophm** [comment]
@harishgilimi please create a new branch for 3.1.0. I have raised the pr to 2.7 as this the latest branch available.

**Comment by HarishGangula** [comment]
@anoophm created release 3.1.0

**Comment by HarishGangula** [comment]
@anoophm  for this PR either test cases or testing is required to see actually connections are closed. please check and let me know this

**Comment by anoophm** [comment]
@harishgilimi there no test case written to cassandra or any db adopters. I have verified this in portal am not getting any errors on close call. There are 4 plugins 4 connections, got four success on server close.

**Comment by HarishGangula** [comment]
ok

**Pull request merged**